Lib/email/message.py

```diff
@@ -318,8 +319,16 @@ def get_payload(self, i=None, decode=False):
                 self.policy.handle_defect(self, defect)
             return value
         elif cte in ('x-uuencode', 'uuencode', 'uue', 'x-uue'):
+            warnings._deprecated(
+                'Support for decoding legacy uuencoded payloads in messages '
+                'is deprecated and scheduled for removal in Python {remove}',
+                remove=(3, 14))
```

Comment:
```suggestion
            warnings._deprecated(
                'Decoding legacy uuencoded payloads in messages'
                remove=(3, 14))
```

Fix deprecation warning call signature